_targets/build.common

```diff
@@ -133,9 +147,17 @@ b_add2img() {
 	: "${OFFSET:?variable unset}"    # starting file's position in the image
 	: "${SIZE_PAGE:?variable unset}"
 	: "${PAGE_MASK:?variable unset}"
+	: "${FLASH_IMAGE:?variable unset}"
```

Comment:
please describe the meaning and either make it an optional with default val or set per-project